api/casauth.go

```diff
@@ -0,0 +1,45 @@
+package api
+
+import (
+	"github.com/wtg/shuttletracker/database"
+	"gopkg.in/cas.v2"
+	"net/http"
+	"net/url"
+	"strings"
+)
+
+// CasClient stores the local cas client and an instance of the database
+type CasClient struct {
```

Comment:
Does this need to be exported? If we're not intending to use it in another package, we should just keep it internal.

Reply:
Should be named `CASClient`